codespan-reporting/src/emitter/views/border_left.rs

```diff
@@ -0,0 +1,23 @@
+use std::io;
+use termcolor::{ColorSpec, WriteColor};
+
+use crate::emitter::Config;
+
+/// The left-hand border of a source line.
+pub struct BorderLeft {}
+
+impl<'a> BorderLeft {
```

Comment:
Why is there a lifetime parameter here?

Reply:
oh, that is a mistake - weird that rustc didn't complain

Reply:
Good catch - it was there in a few of them - copy paste issue I think